test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,31 +261,84 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100" }, hostedElementsIncludeNothing);
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeEverything);
+
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeOpenings);
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeOpeningsAndShadows);
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeOpeningsAndShadowsAndEmbeddedWalls);
+
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100"}, hostedElementsIncludeShadows);
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100" }, hostedElementsIncludeShadowsAndEmbeddedWalls);
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100" }, hostedElementsIncludeShadowsAndEmbeddedWallsAndEmbeddedInserts);
+
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100" }, hostedElementsIncludeEmbeddedWalls);
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100" }, hostedElementsIncludeEmbeddedWallsAndEmbeddedInserts);
+            AssertExpectedElementNameCollection(new[] { "Curtain Wall", "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeEmbeddedWallsAndEmbeddedInsertsAndOpenings);
+
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100" }, hostedElementsIncludeEmbeddedInserts);
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeSEmbeddedInsertsAndOpenings);
+            AssertExpectedElementNameCollection(new[] { "600 x 3100", "600 x 3100", "600 x 3100", "Rectangular Straight Wall Opening" }, hostedElementsIncludeEmbeddedInsertsAndOpeningsAndShadows);
+            #endregion
         }
 
-        [Test]
-        [TestModel(@".\element.rvt")]
-        public void CanSuccessfullyGetHostedElements()
+        private static void AssertExpectedElementNameCollection(string[] expectedNameCollection, List<Element> hostedElementCollection)
         {
-            var wall = ElementSelector.ByElementId(184176, true);
-            GetExpectedWallHeight(wall, 20);
+            CollectionAssert.AreEqual(expectedNameCollection, hostedElementCollection.Select(x => x.Name).ToArray());
```

Comment:
since our method only has 1 method, it'd be clearer to just use the `CollectionAssert...` code, let's get rid of this method and use that code directly.